node/src/actors/peers_manager/mod.rs

```diff
@@ -5,6 +5,7 @@ use actix::{
     SystemService, WrapFuture,
 };
 
+use crate::actors::messages::RemoveAddressesFromTried;
 use crate::{
     actors::{connections_manager::ConnectionsManager, messages::OutboundTcpConnect, storage_keys},
```

Comment:
Merge this import and run `just fmt`:

```suggestion
use crate::{
    actors::{connections_manager::ConnectionsManager, messages::{OutboundTcpConnect, RemoveAddressesFromTried}, storage_keys},
```